include/nbl/video/utilities/IUtilities.h

```diff
@@ -260,9 +259,8 @@ class NBL_API IUtilities : public core::IReferenceCounted
             }
 
             const auto finalLayout = params.initialLayout;
-
-            if (!((params.usage & asset::IImage::EUF_TRANSFER_DST_BIT).value))
-                params.usage |= asset::IImage::EUF_TRANSFER_DST_BIT;
+            
+            assert(params.usage.hasFlags(asset::IImage::EUF_TRANSFER_DST_BIT));
```

Comment:
again don't assert, just return nullptr, we'll have matching errors in Release and Debug